pkg/infra/iac2/templates/ecr_image/package.json

```diff
@@ -3,6 +3,7 @@
     "dependencies": {
         "@pulumi/aws": "^5.16.0",
         "@pulumi/pulumi": "^3.40.2",
-        "@pulumi/docker": "^v4.1.0"
+        "@pulumi/docker": "^v4.1.0",
+        "@types/node": "^18.15.11"
```

Comment:
Is this used?

Reply:
yeah we need it for the Buffer type even though its not imported

Reply:
i guess our dep check says we dont but let me see what happens if i remove it